scipy/linalg/setup.py

```diff
@@ -118,7 +118,7 @@ def configuration(parent_package='', top_path=None):
 
 if __name__ == '__main__':
     from numpy.distutils.core import setup
-    from linalg_version import linalg_version
+    from scipy.linalg.linalg_version import linalg_version
```

Comment:
I think this is equivalent? (And, if so, seems like a better practice?)

Reply:
Yes this relative import business is a mess actually. See this post for more info https://stackoverflow.com/a/28154841